### DISCUSSION TITLE
Getting sensitivity map or Fresnel weights map

### DISCUSSION THREAD

**Discussion by makeabhishek** [gimli-org/pyGIMLi#563] [General]
Hi,
How can we obtain the Fresnel weights as shown in below figure using #Pygimli in traveltime tomography? 
![image](https://github.com/gimli-org/gimli/assets/47937684/c1605542-66e2-4ce6-a476-a2cc741c5bcc)

Ref: Gance, Julien, et al. "Quasi-Newton inversion of seismic first arrivals using source finite bandwidth assumption: Application to subsurface characterization of landslides." Journal of Applied Geophysics 87 (2012): 94-106.

**Comment by halbmy** [comment]
I've made some modifications to the Fatray operator so that it is now working again, however, not yet with secondary nodes which is the next step before I can push it.

I've made a notebook using the normal and the fatray operator and also showing the Fresnel weights, based on the example field data set:
https://github.com/gimli-org/notebooks/blob/main/traveltime/fatray.ipynb

Here's the Fresnel weight image again:

![image](https://github.com/gimli-org/gimli/assets/7136401/d85a49e3-4011-4c2a-a7e3-59cdef81f04a)
